src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/OffenderMatchCriteria.kt

```diff
@@ -16,9 +16,9 @@ data class OffenderMatchCriteria(
   companion object {
     fun from(person: Person): OffenderMatchCriteria {
       return OffenderMatchCriteria(
-        firstName = person.givenName,
-        surname = person.familyName,
-        dateOfBirth = person.dateOfBirth,
+        firstName = person.names.preferred?.firstName,
+        surname = person.names.preferred?.lastName,
+        dateOfBirth = person.names.preferred?.dateOfBirth,
```

Comment:
I'm a bit confused by the nullable here - surely if we don't have a preferred name we don't have a record?